packages/tasks/src/local-apps.ts

```diff
@@ -389,6 +399,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
 	},
+	ollama: {
+		prettyLabel: "Ollama",
+		docsUrl: "https://ollama.com",
+		mainTask: "text-generation",
+		displayOnModelPage: (model) => model.tags.includes("conversational") && isLlamaCppGgufModel(model),
```

Comment:
rm conversational no?

Reply:
handled in cbf44d6165284d14d74a4bb644278032176967ef as ollama also supports embedding & next-token prediction models (besides the conversational ones)